Author: Pino Toscano <pino@debian.org>
Description: workaround pthread issues on GNU/Hurd
 Explicit link the main ggadget library to pthread, so we can make sure to
 have the right pthread loaded (instead of pthread-stubs).
 This is temporary anyway, will be solved when Hurd's libpthread will be
 integrated in glibc.
Forwarded: not-needed
--- a/ggadget/Makefile.am
+++ b/ggadget/Makefile.am
@@ -253,6 +253,7 @@
 			  $(DEFAULT_COMPILE_FLAGS)
 
 libggadget@GGL_EPOCH@_la_LIBADD = \
+			  $(PTHREAD_LIBS) \
 			  $(top_builddir)/third_party/unzip/libunzip.la
 
 libggadget@GGL_EPOCH@_la_LDFLAGS = \
